<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewPatientWindow</class>
 <widget class="QWidget" name="NewPatientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add New Patient</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="6">
    <widget class="QRadioButton" name="input_dog">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Dog</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="food_checkbox">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Food:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="8">
    <widget class="QPushButton" name="push">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Push</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Patient:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="7">
    <widget class="QPlainTextEdit" name="input_treatment"/>
   </item>
   <item row="4" column="2" colspan="7">
    <widget class="QPlainTextEdit" name="input_medication"/>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QCheckBox" name="leash_checkbox">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Leash</string>
     </property>
    </widget>
   </item>
   <item row="9" column="7">
    <widget class="QLineEdit" name="input_device">
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2" alignment="Qt::AlignTop">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Medication:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" alignment="Qt::AlignTop">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Treatment:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QCheckBox" name="toys_checkbox">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Toys</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="4">
    <widget class="QCheckBox" name="bed_blanket_checkbox">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Bed/Blanket</string>
     </property>
    </widget>
   </item>
   <item row="9" column="5" colspan="2" alignment="Qt::AlignRight">
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Device:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="4">
    <widget class="QLineEdit" name="input_patient">
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QRadioButton" name="input_cat">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Cat</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="4">
    <widget class="QLineEdit" name="input_owner"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Owner:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Amount:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QLineEdit" name="input_food"/>
   </item>
   <item row="5" column="5" colspan="2">
    <widget class="QLineEdit" name="input_food_amount"/>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="carrier_checkbox">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Carrier</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
